app/controllers/groupings_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class GroupingsController < ApplicationController
+  include OrganizationAuthorization
+
+  before_action :set_grouping
+
+  def show
+    not_found unless Classroom.flipper[:team_management].enabled? current_user
+  end
+
+  private
+
+  def set_grouping
+    @grouping = Grouping.find_by!(id: params[:id])
```

Comment:
Should use slug here. I'm going to create a migration and make `grouping` sluggable.